packages/foundation/lib/schema.js

```diff
@@ -49,6 +49,31 @@ export const workers = {
   ]
 }
 
+const verticalScaler = {
```

Comment:
We need to have two specification for this. One that goes in runtime, and one that goes in each capability.